client/web/src/hooks/auth.ts

```diff
@@ -16,24 +16,26 @@ export type AuthResponse = {
 // for the web client.
 export default function useAuth() {
   const [data, setData] = useState<AuthResponse>()
-  const [loading, setLoading] = useState<boolean>(false)
+  const [loading, setLoading] = useState<boolean>(true)
 
   const loadAuth = useCallback((wait?: boolean) => {
     const url = wait ? "/auth?wait=true" : "/auth"
-    setLoading(true)
```

Comment:
I think we still want this - if `loadAuth` gets triggered again, we'd want to set it.

Reply:
ah yes.  done